README.md

```diff
@@ -1 +1,298 @@
-# guesser-game-api
+# Guesser-Game-API
+
+## What is Guesser Game API?
+
+Guesser Game Api é um jogo, com ele é possível cadastrar uma palavra secreta com três dicas, a cada tentativa incorreta o contado de tentativas é agressido de um. Ao atingir os valores pré-definidos em clue1_attempts, clue2_attempts e clue3_attempts, os valores refetentes a clue1, clue2 e clue3 são revelados. Quando o palpite estiver correto, o valor do segredo é revelado.
+
+## Project Structure
+
+### Handlers
+
+The Guesser Game API includes useful handlers for daily needs. Below, we list and provide examples for each of them.
+
+#### Utils
+
+**health_handler**
+
+This handler is used to check the health of the API. Make a GET request to the URL/health. If the API is functioning correctly, a status 200 and a JSON will be returned, as shown below:
+
+```json
+{
+  "status": "pass"
+}
+```
+
+**full_health_handler**
+
+This handler is used to check the health of both the API and the database. Make a GET request to URL/health/full. Depending on the situation, it will return a status 200 or an error, accompanied by a JSON. Examples:
```

Comment:
Same as above:

```suggestion
This handler is used to check the health of both the API and the database. Make a GET request to URL `/health/full`. Depending on the situation, it will return a status 200 or an error, accompanied by a JSON. Examples:
```